src/emailcontact.h

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) 2020- 2021 Open Mobile Platform LLC.
+ *
+ * This program is licensed under the terms and conditions of the
+ * Apache License, version 2.0.  The full text of the Apache License is at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+#ifndef EMAILCONTACT_H
+#define EMAILCONTACT_H
+
+#include <QMailAddress>
+#include <QMailTimeStamp>
+
+
+class Q_DECL_EXPORT EmailContact
+{
+public:
+    explicit EmailContact() = default;
+    explicit EmailContact(const QString &name, const QString &email, const QMailTimeStamp &timeStamp);
+    explicit EmailContact(const QMailAddress &address, const QMailTimeStamp &timeStamp);
+
+    QString name() const;
+    QString email() const;
+    QMailAddress address() const;
+    QMailTimeStamp timeStamp() const;
+
+    void setName(const QString &name);
+    void setEmail(const QString &email);
+    void setAddress(const QMailAddress &address);
```

Comment:
Out of this group only the below timestamp seems to be used. -> could remove everything else. Could be even an option to remove the timestamp and just do new instances per need.

Reply:
fixed it